key,default_value
userAgent,"aws-cli/1.14.44 Python/3.6.9 Linux/5.3.0-51-generic botocore/1.8.48"
eventID,"39b68aa6-4805-4fc8-af84-0381cd75282b"
errorMessage,"User: arn:aws:iam::811596193553:user/backup is not authorized to perform: rds:RestoreDBInstanceFromDBSnapshot on resource: arn:aws:rds:us-west-2:811596193553:db:vol-8e100f305b7a6fef3"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"251.63.7.255"
eventName,"RestoreDBInstanceFromDBSnapshot"
eventSource,"rds.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-west-2"
requestID,"0d2cceb2-8bfe-48b5-8011-c8026be025d5"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2020-05-16T20:31:50Z"
